e2e/playwright/flow-tests.spec.ts

```diff
@@ -0,0 +1,395 @@
+import { test, expect } from '@playwright/test'
+import { secrets } from './secrets'
+import { EngineCommand } from '../../src/lang/std/engineConnection'
+import { v4 as uuidv4 } from 'uuid'
+import { getUtils } from './test-utils'
+
+/*
+debug helper: unfortunately we do rely on exact coord mouse clicks in a few places
+just from the nature of the stream, running the test with debugger and pasting the below
+into the console can be useful to get coords
+
+document.addEventListener('mousemove', (e) =>
+  console.log(`await page.mouse.click(${e.clientX}, ${e.clientY})`)
+)
+*/
+
+test.beforeEach(async ({ context, page }) => {
+  context.addInitScript(async (token) => {
+    localStorage.setItem('TOKEN_PERSIST_KEY', token)
+    localStorage.setItem('persistCode', ``)
```

Comment:
@Irev-Dev alright so this is how you handle auth in web tests? Cool that it's a more direct mechanism compared to #1040